tests/raftstore/server.rs

```diff
@@ -121,7 +121,12 @@ impl Simulator for ServerCluster {
 
         // Create pd client, snapshot manager, server.
         let (worker, resolver) = resolve::new_resolver(self.pd_client.clone()).unwrap();
-        let snap_mgr = SnapManager::new(tmp_str, Some(store_sendch));
+        let limiter = Arc::new(SnapshotIOLimiter::new(
```

Comment:
can you support the default function for the SnapshotIOLimiter so you don't need to pass the similar args in test every time?